core/src/main/java/hudson/Functions.java

```diff
@@ -1936,6 +1936,13 @@ public boolean hyperlinkMatchesCurrentPage(String href) {
         return url.endsWith(href);
     }
 
+    /**
+     * @deprecated From JEXL expressions ({@code ${…}}) in {@code *.jelly} files
+     *             you can use {@code [obj]} syntax to construct an {@code Object[]}
+     *             (which may be usable where a {@link List} is expected)
+     *             rather than {@code h.singletonList(obj)}.
+     */
+    @Deprecated
```

Comment:
```suggestion
    @Deprecated(since = "TODO")
```

Reply:
Not sure why it would matter. It could be considered retroactively deprecated since the beginning of the Hudson project. Anyway the call sites are in Jelly sources where no IDE will be able to display this information.